integration/integration_test.go

```diff
@@ -469,10 +544,32 @@ func TestExecsnoop(t *testing.T) {
 	t.Parallel()
 
 	execsnoopCmd := &command{
-		name:           "StartExecsnoopGadget",
-		cmd:            fmt.Sprintf("$KUBECTL_GADGET trace exec -n %s", ns),
-		expectedRegexp: fmt.Sprintf(`%s\s+test-pod\s+test-pod\s+\d+\s+\d+\s+date`, ns),
-		startAndStop:   true,
+		name:         "StartExecsnoopGadget",
+		cmd:          fmt.Sprintf("$KUBECTL_GADGET trace exec -n %s -o json", ns),
+		startAndStop: true,
+		customOutputVerifier: func(output string) error {
+			expected := execTypes.Event{
+				Event: buildBaseEvent(ns),
+				Comm:  "date",
+				Args:  []string{"/bin/date"},
+			}
```

Comment:
Another cool thing about this change + this approach is that we could verify multiple entries. For instance, for the trace `exec` gadget:
 
1.` ..."pid":23861,"ppid":23841,"mountnsid":4026532375,"pcomm":"sh","args":["/bin/sh","-c","while true; do date \u0026\u0026 sleep 0.1; done"]}`
2. `... "pid":23877,"ppid":23861,"mountnsid":4026532375,"pcomm":"date","args":["/bin/date"]}`
3. `... "pid":23878,"ppid":23861,"mountnsid":4026532375,"pcomm":"sleep","args":["/bin/sleep","0.1"]}`

Currently, I just added the one for `date`, but I will add the other two.